code/game/objects/structures/toiletbong.dm

```diff
@@ -93,9 +94,12 @@
 	if (!do_after(user, 10 SECONDS, target = src))
 		return FALSE
 	new /obj/item/flamethrower(get_turf(src))
-	new /obj/item/stack/sheet/iron(get_turf(src))
 	var/obj/item/tank/internals/plasma/ptank = new /obj/item/tank/internals/plasma(get_turf(src))
 	ptank.air_contents.gases[/datum/gas/plasma][MOLES] = (0)
+	for(var/datum/material/material as anything in custom_materials)
+		var/sheets_to_spawn = FLOOR(custom_materials[material] / SHEET_MATERIAL_AMOUNT, 1)
+		if(sheets_to_spawn > 0 && material.sheet_type)
+			new material.sheet_type(loc, sheets_to_spawn)
```

Comment:
Tempted to say move this into a generic proc, if only because I can think of a few things that may want a generic "deconvert custom_mats into component sheets" proc, plus it's reused for regular toilet deconstruction a few files down